packages/persistent-merkle-tree/src/proof/index.ts

```diff
@@ -12,6 +13,7 @@ import {
 export enum ProofType {
   single = "single",
   treeOffset = "treeOffset",
+  EFMulti = "EFMulti",
```

Comment:
Then should we find a better name that's not tied to an organization? Some name that's generic to the data structure like "testOffset"